driver/normalizer/normalizer.go

```diff
@@ -87,14 +87,14 @@ var Normalizers = []Mapping{
 		CasesObj("case",
 			// common
 			Obj{
-				"importKind": String("value"),
+				"importKind": Var("kind"),
```

Comment:
I don't see the matching variable on the other side. The transformation will drop this field.

Same for `spec_type` below.

Reply:
It's ignored, just like it's not added to the other side on the previous version (just checked for a certain value), I just want to take any value since all the internal types I saw used in this field had the same structure, is there a better way to do this?

Reply:
But you are losing the information from the native AST. Are you sure all these nodes are semantically the same, even having the same structure? I think of it like dropping the `private` in Java. This shouldn't happen without a very good reason.

In any case, a variable is not the right way to do it. `AnyNode(nil)` would be better.

Reply:
I added the specific "kind" types for `ImportDeclaration` in the `CaseObj` in the last commit PTAL.